lib/errors.ts

```diff
@@ -247,6 +247,24 @@ export class BalenaAmbiguousApplication extends BalenaError {
 }
 BalenaAmbiguousApplication.prototype.code = 'BalenaAmbiguousApplication';
 
+/**
+ * @summary Balena ambiguous release
+ * @class
+ * @public
+ *
+ * @param {(String)} release - release hash
+ * @return {Error} error instance
+ *
+ * @example
+ * throw new errors.BalenaAmbiguousRelease('7cf02a6')
+ */
+export class BalenaAmbiguousRelease extends BalenaError {
+	constructor(public release: string) {
+		super(`Release is ambiguous: There are multiple releases matching the release hash '${release}'`);
+	}
+}
+BalenaAmbiguousRelease.prototype.code = 'BalenaAmbiguousRelease';
```

Comment:
Reading issue https://github.com/balena-io/balena-cli/issues/1064, it sounds like what is ambiguous is the commit, not the release... I am happy either way, but perhaps `BalenaAmbiguousCommit` would be more precise than `BalenaAmbiguousRelease`.

Reply:
If I get this right, it's more like that the resourceX items (in this case release) that we ended up to as a result of filtering based on the parameter that the user provided are more than one, in other words:
> Dear user, it was ambiguous which release you meant with the parameters that you provided.

Reply:
Yes, you're right, the user's release specification is ambiguous. 👍